src/components/EventCard.astro

```diff
@@ -60,7 +60,7 @@ const hoverTextShadowColor =
         set:html={date}
         class={classNames(
           "card-date",
-          "text-center md:text-base text-gray-200 tracking-wider mt-8",
+          "text-center md:text-base text-gray-200 tracking-wider mt-8 sm:mt-4 md:mt-8",

```

Comment:
I am not sure if this is what you want to achieve. Just to make sure: When you assign `mt-8 sm:mt-4`, the `margin-top` will be `8` on devices `<640px`, then `4` on devices greater than that. I assume you wanted it to increase relatively with the screen size

Reply:
No, this is actually indented. On very small devices, these cards are shown in a column. In the range in between, they are shown in a row, and on larger devices they're shown in the layout. It's not a great solution, but it works for now, and I wanted to get this merged before everyone is done registering